dashboards/workspace/workspace_dashboard.sp

```diff
@@ -13,140 +13,63 @@ dashboard "workspace_dashboard" {
     title = "Workspace Statistics"
 
     card {
-      sql   = query.workspace_count.sql
-      width = 3
+      sql   = query.workspaces_count.sql
+      width = 2
```

Comment:
@vkumbha I think the card width should be `3`. Is there any particular reason why we changed it to `2`?

Reply:
You are right. updated the width to 3.